### PR TITLE
Added : Cart and Wishlist Page

### PR DESCRIPTION
**Pull request by devsurajrai**
Added Cart and Wishlist Page. Please review the PR. Files/directory to review is mentioned for more convenience.
- /CartMain.component
- /WishlistMain.component
- /pages/Cart.jsx
- /pages/Wishlist.jsx

Screenshot 2022-03-20 at 5 45 08 PM
Screenshot 2022-03-20 at 5 44 57 PM

### CODE REVIEW

File: src/components/header.component/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          <button className="button button--secondary login-btn">
+            <a href="./log-in.html">Login</a>
+          </button>
+        </li>
+        <li>
+          <a
+            className="color-white text-sm"
+            href="../product-listing/product-listing.html"
+          >
+            SHOP NOW{" "}
```
Comment (Dev-AkshatGupta) [inline src/components/header.component/Header.jsx:36]: Remove the empty jsx component from this line

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems cool to me you can go ahead with this

File: src/components/header.component/Header.jsx (hunk 1 above)
Comment (devsurajrai) [reply src/components/header.component/Header.jsx:36]: Thanks ! Will do that.

File: src/components/HomeMain.component/Hero.component/Hero.jsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const Hero = () => {
+  return (
+    <img
+      className="res-img br-sm"
+      src="https://ui-phoenix.netlify.app/Assets/images/lee-chong-wei.jpeg"
+      alt=""
```
Comment (sushmita12345) [inline src/components/HomeMain.component/Hero.component/Hero.jsx:6]: Suggest to fill the alt="" attribute

File: src/components/HomeMain.component/Categories.component/HomeCategories.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import racket from "../../../Assets/images/racquet.png";
+import React from "react";
+import "./home-categories.css";
+
+const HomeCategories = () => {
+  return (
+    <div className="cat-card br-sm">
+      <img className="res-img" src={racket} alt="" />
```
Comment (sushmita12345) [inline src/components/HomeMain.component/Categories.component/HomeCategories.jsx:8]: Here also fill the alt="" attribute

**Review by sushmita12345 — APPROVED** [review]

File: src/components/HomeMain.component/Hero.component/Hero.jsx (hunk 2 above)
Comment (devsurajrai) [reply src/components/HomeMain.component/Hero.component/Hero.jsx:6]: Will do that. Thanks

File: src/components/HomeMain.component/Categories.component/HomeCategories.jsx (hunk 3 above)
Comment (devsurajrai) [reply src/components/HomeMain.component/Categories.component/HomeCategories.jsx:8]: Thanks . Will do that

**Pull request merged**